src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java

```diff
@@ -36,20 +30,21 @@ public class GenomicEntity extends BiologicalEntity {
 
 	@IndexedEmbedded(includePaths = {"referencedCurie", "displayName", "referencedCurie_keyword", "displayName_keyword"})
 	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-	@ManyToMany
+	@OneToMany(cascade = CascadeType.ALL, orphanRemoval=true)
```

Comment:
So this relationship type should be ManyToMany given that there is a join table. Which does not support the orphanRemoval. So typically what would be done here is another class is created: GenomicEntityCrossReference which represents the entry in the join table. Then this class has a List<GenomicEntityCrossReference> with a OneToMany relationship.

Reply:
Would that not be a departure from the LinkML model though?  So should we just go back to manual cleanup of xrefs?

Reply:
It jsut that via hibernate a OneToMany would imply that there is a table called CrossReference that has a GenomeEntityFK back to this table... that is not true in that there is a join table in between the two tables which really should be considered as a ManyToMany relationship. This works, but is not the ideal hibernate way of doing it.